resources/zib/zib-AbilityToDrink.xml

```diff
@@ -0,0 +1,122 @@
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="AbilityToDrink" />
+      <alias value="VermogenTotDrinken" />
+      <mapping>
+        <identity value="zib-abilitytodrink-v3.1.1-2020EN" />
+        <map value="NL-CM:4.8.1" />
+        <comment value="AbilityToDrink" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="288852001" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueCodeableConcept">
+      <path value="Observation.valueCodeableConcept" />
+      <sliceName value="valueCodeableConcept" />
+      <short value="Drinking" />
+      <definition value="Taking fluids during meals and during the day or when thirsty." />
+      <alias value="Drinken" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.4.8.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-abilitytodrink-v3.1.1-2020EN" />
+        <map value="NL-CM:4.8.3" />
+        <comment value="Drinking" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:drinkingLimitations">
+      <path value="Observation.component" />
+      <sliceName value="drinkingLimitations" />
+    </element>
+    <element id="Observation.component:drinkingLimitations.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="288851008" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:drinkingLimitations.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="DrinkingLimitations" />
+      <definition value="DrinkingConstraints specifies the patient’s constraints in drinking." />
```

Comment:
Replace 'DrinkingConstraints' by 'DrinkingLimitations' (and for consistency also 'constraints' by 'limitations'). I'll create a zib ticket to change this in the zib itself as well ('DrinkingConstraints' was the name of this element in the 2015/2016 version).

Reply:
We add this stuff automatically from the zib definition, so we don't manually edit these in our profiles. This is the responsibility of the zib-centrum, so creating a zib ticket to have this corrected is the way to go (although there are so much of these kinds of errors that I wonder when you'll get tired of it).